README.md

```diff
@@ -346,6 +346,8 @@ Here is a [CSS in JS techniques comparison](https://github.com/MicheleBertoli/cs
 
 * [CSS Secrets](http://shop.oreilly.com/product/0636920031123.do) – Better Solutions to Everyday Web Design Problems
 * [CSS: The Missing Manual](http://shop.oreilly.com/product/0636920036357.do) – Really Helpful in Advancing your Design Skills to a whole new Level
+* [CSS in Depth](https://www.manning.com/books/css-in-depth) - CSS in Depth takes web developers from beginner to advanced CSS users, getting rid of their blind spots along the way.
```

Comment:
Here we're looking for books that can be purchased (ebook + hardcopy), but the publication of this book has not started yet, so this can't be added here at this time 😞 , you can suggest some other books though, that will be highly encouraged.